app/celery/research_mode_tasks.py

```diff
@@ -149,7 +149,7 @@ def ses_notification_callback(reference):
             "processingTimeMillis": 2003,
             "recipients": ["success@simulator.amazonses.com"],
             "remoteMtaIp": "123.123.123.123",
-            "reportingMTA": "a7-32.smtp-out.eu-west-1.amazonses.com",
+            "reportingMTA": "a7-32.smtp-out.eu-west-2.amazonses.com",
```

Comment:
Ooh, good spot

Reply:
Endpoints and ARNs associated with SES will be corrected when we work on the SES integration ticket. For now, we didn't want to leave any references to 'eu-west-1' in the codebase.